vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -84,6 +84,9 @@ type QMPConfig struct {
 	// logger is used by the qmpStart function and all the go routines
 	// it spawns to log information.
 	Logger QMPLog
+
+	// specify the capacity of buffer used by receive QMP response.
+	MaxCapacity int
```

Comment:
Should this be unsigned to avoid surprises?

Reply:
`max` is defined as `int` in https://golang.org/pkg/bufio/#Scanner.Buffer, so i keep the type here